File: daySix/Python/sorted_array.py
def new_array_length(array, newlength):
    new_array = [0] * newlength

    for count in range(min(newlength, len(array))):
        new_array[count] = array[count]

    return new_array


def print_array(array):

    print("[", end="")

    for count in range(len(array)):
    
        print(array[count], end="")

        if count != len(array) - 1:
        
            print(", ", end="")

    print("]")


def is_prime(number):

    if number <= 1:
    
        return False

    for count in range(2, number):
    
        if number % count == 0:
        
            return False

    return True


def sort_the_array(array):

    for count in range(len(array) - 1):
    
        for check in range(len(array) - 1 - count):
        
            if array[check] < array[check + 1]:
            
                temp = array[check]
                
                array[check] = array[check + 1]
                
                array[check + 1] = temp


def sorted_primes(array):

    temp = [0] * len(array)

    counter = 0

    for count in range(len(array)):
    
        if is_prime(array[count]):
        
            temp[counter] = array[count]
            
            counter += 1

    prime_numbers = new_array_length(temp, counter)

    sort_the_array(prime_numbers)

    return prime_numbers

File: daySix/Java/SortedArray.java
import java.util.Arrays;

public class SortedArray{

    public static int[] newArrayLength(int[] array, int newlength){
    
        int[] newArray = new int[newlength];
        
        for (int count = 0; count < newlength && count < array.length; count++){
        
            newArray[count] = array[count];
               
        }
        return newArray;
    }
    
    
    public static void printArray(int[] array){
    
        System.out.print("[");
        
        for (int count = 0; count < array.length; count++){
        
            System.out.print(array[count]);
            
            if (count != array.length - 1){
            
                System.out.print(", ");
            }   
        
        }
        System.out.println("}");    
    
    }
    
    public static boolean isPrime(int number){
    
        if(number <= 1) {
        
		return false;
		
		}

		for(int count = 2; count < number; count++) {
		
			if(number % count == 0) {
			
				return false;
			}
		}
		return true;
	}
	
	
	
	public static void sortTheArray(int[] array){
		       
	    for(int count = 0; count < array.length-1; count++){
    
            for(int check = 0; check < (array.length-1) - count; check++){

                if(array[check] < array[check + 1]){

                    int temp = array[check];

                    array[check] = array[check + 1];

                    array[check + 1] = temp;
    
                 }


            }

        }
            
       
    }
    
    
    public static int[] sortedPrimes(int[] array){
    
        int[] temp = new int[array.length];
        
        int counter = 0;
        
        for(int count = 0; count < array.length; count++){
        
            if(isPrime(array[count])){
            
                temp[counter] = array[count];
                
                counter++;
            
            }
        
        
        }
    
        int[] primeNumbers = newArrayLength(temp, counter);
        
        sortTheArray(primeNumbers);
        
        
        return primeNumbers;
     
    }
    
    
    
    public static void main(String[] args){
    
    int[] array = {5, 9, 3, 6, 2};
    
    System.out.println(Arrays.toString(sortedPrimes(array)));
    
    
    }
    
    
    
    
    
    
    
    }
    
    
    
